snappy/handlers.go

```diff
@@ -147,3 +168,34 @@ func (h *Handler) MakeMuxer(prefix string) http.Handler {
 
 	return m
 }
+
+// TODO: refactor this copy from cmd/snapweb
```

Comment:
because that code is in package 'main', i can't import it; otherwise it takes moving around quite a bit of code, like the token, or the cookie code; which i'd rather do in a future branch